.release-notes/4147.md

```diff
@@ -0,0 +1,14 @@
+## Change error message
+
+The error message when assigning to a field of a field in box functions was fixed to reflect that the function should be a fun ref. This affects code like the below:
+
+```pony
+actor Main
+        let my_test_class: TestClass= TestClass
+        new create(env: Env) =>
+                """placeholder"""
+        fun test() =>
+                my_test_class.x= 1
```

Comment:
any code included in release notes as an example should be able to compile. This code isn't complete and therefore won't compile and doesn't demonstrate what might have been the issue.

Reply:
sorry, i updated it